apps/consumer-client/src/pages/examples/gpc-proof.tsx

```diff
@@ -195,13 +195,26 @@ export function openGPCPopup(
       value: proofConfig,
       userProvided: false
     },
-    pod: {
-      argumentType: ArgumentTypeName.PCD,
-      pcdType: PODPCDPackage.name,
-      value: undefined,
-      userProvided: true,
+    pods: {
+      argumentType: ArgumentTypeName.Record,
+      value: {
+        mainstreamPOD: {
+          argumentType: ArgumentTypeName.PCD,
+          pcdType: PODPCDPackage.name,
+          value: undefined,
+          userProvided: true,
+          displayName: "Mainstream POD"
+        },
+        ticketPOD: {
+          argumentType: ArgumentTypeName.PCD,
+          pcdType: PODPCDPackage.name,
```

Comment:
It would be awesome if this could be a PODTicketPCD and "just work".
Seems a relatively easy extension, but doesn't need to be part of this PR.